01-04-2024/src/js/exercises.js

```diff
@@ -0,0 +1,166 @@
+import { contacts } from '../data/contacts.js';
+
+//console.log(contacts);
+/**Array Order Methods */
+
+/**
+ * Function that takes the array of contacts and returns a new array sorted by last name.
+ * @param {array} arrayContacts array of contacts
+ * @returns {array} array of contacts
+ */
+const getSortedByLast = (arrayContacts) => {
+  const result = [...arrayContacts].sort((contact1, contact2) =>
```

Comment:
good way to avoid mutation by shallow clone
consider using deep clone to avoid any risk of mutating source